src/styled/components/Buttons.js

```diff
@@ -66,8 +68,10 @@ export const Avatar = Button.extend`
   padding: 0;
   position: relative;
   width: 40px;
+
   span {
     left: 50%;
     position: absolute;
-    transform: translateX(-50%);
+    tran: translateX(-50%);
```

Comment:
Possibly a mistake?

Reply:
yes! and fixed, no idea how I did that...